my-sms/app/assets/stylesheets/institutes.css.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the institutes controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```

Comment:
File can be removed

Reply:
Fixed in 67792896a31da920aff3f1f48a47966c3180d376